src/iterative_sorting/iterative_sorting.py

```diff
@@ -6,21 +6,29 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
+        for j in range(cur_index + 1, len(arr)):
+            if arr[cur_index] > arr[j]:
+                smallest_index = arr[j]
         # TO-DO: swap
-
-
-
+                arr[j] = arr[cur_index]
+                arr[cur_index] = smallest_index
 
     return arr
 
 
+
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort( arr ):
-
+    sorting = True
+    while(sorting):
+        sorting = False
```

Comment:
good job turning off that sorting boolean here. remember how you did this; will be useful on friday!